src/client/lib/downloadInstaller.js

```diff
@@ -37,21 +37,21 @@ Make sure the files are available on their respective locations when distributin
 `,
   )
 
-  const installerDownloads = installersData.map(({ branch, manifestName })=> {
-    const winInstaller = createInstaller(branch, manifestName)
-    const osxInstaller = createInstaller(branch, manifestName, 'osx') 
-    return Promise.all([winInstaller, osxInstaller])
+  const installerDownloads = installersData.map(({ branch, manifestName, os })=> {
+    return createInstaller(branch, manifestName, os)
   });
 
   return Promise.all(installerDownloads)
 }
 
 // File name + github branch for each json manifest
 const INSTALLERS_TO_CREATE = [
-  { manifestName: 'dev', branch: 'develop' },
-  { manifestName: 'launcher-dev', branch: 'develop' },
-  { manifestName: 'demo', branch: 'master' },
-  { manifestName: 'launcher-demo', branch: 'master' },
+  { manifestName: 'dev', branch: 'master', os: 'win' },
```

Comment:
is it a bit odd that we crate two installers (dev AND demo) from the same branch ?

Reply:
i thought that dev and demo relate to the branches..but maybe they are not :/ confused

Reply:
ah, everything is clear now (after looking at actual manifests)